### PR TITLE
 massive commit: detect outdated/obscure mods (stuff from at least a year ago), slightly more instances of XYZs (iykyk), and crackdown on flipping mods in case one of them crases since skyclient doesnt provide support for those types of mods (correct me if im wrong here)

### PR DESCRIPTION
**Pull request by RayDeeUx**
lmk which edits i should make

### CODE REVIEW

**Comment by RayDeeUx** [comment]
yes i spent most of today manually checking 56 pages of repositories to see which were mc mods and what offense they deserved

proaction momentum

**Review by MicrocontrollersDev — APPROVED** [review]
bro needs a hobby wtf

**Comment by RayDeeUx** [comment]
you fools

this is a hobby

**Comment by RayDeeUx** [comment]
i guess now's a better time than never to say that i originally had more mods to add to the obscure category but didn't want to be TOO aggressive, especially to "mod" repos with zero java files to begin with (yes, hence those airquotes)

**Comment by RayDeeUx** [comment]
also hael9 detection remains ever impossible 😔

**Review by Zetvue — APPROVED** [review]

**Comment by nacrt** [comment]
remove the skytils version detection

**Review by UnderscoreRayless — CHANGES_REQUESTED** [review]
- Line 891, no capitalization needed
- Line 1356, please remove your ery humour
- Line 1400 to line 1559; Rephrase to "Auction flipping mods are unsupported and any issues caused by them will not be investigated further`. The mere existence of the mod doesn't indicate a crash and can and will confuse people, blaming their crash on that mod.
- Line 1770, MacroModModules is a RAT, not a cheat most of the time. Specify that.
- Line 1823, remove the v1.2.5 check.

**Comment by RayDeeUx** [comment]
@UnderscoreRayless @nacrt happy?

File: crashes.json [hunk 1]
```diff
@@ -1820,7 +1791,7 @@
     },
     {
       "fixtype": 1,
-      "fix": "Issue with Skytils. Make sure it's up to date (preferably v1.2.5).",
+      "fix": "Issue with Skytils. Make sure it's up to date (preferably whichever version SkyClient staff want you to be on).",
```
Comment (KTibow) [inline crashes.json:1794]: isn't crash(patch|helper idk) also used outside of skyclient

**Review by KTibow — CHANGES_REQUESTED** [review]

**Comment by UnderscoreRayless** [comment]
> @UnderscoreRayless @nacrt happy?

No. 

- Line 1327 REMOVE THE FUCKING ERY HUMOUR
- Line 1742 Please be sane. No need to freak users out. Just tell them that they likely installed a rat and that they should change their passwords. Also no emojis, this is used in CrashPatch
- Line 1794 Remove the bracket sentence all together.

**Comment by UnderscoreRayless** [comment]
Also please make sure to revert the "ok fine maybe i can be a bit more stict" commit, unnecessary

File: crashes.json [hunk 2]
```diff
@@ -1274,7 +1791,7 @@
     },
     {
       "fixtype": 1,
-      "fix": "Issue with skytils, make sure it's up to date.",
+      "fix": "Issue with Skytils. Make sure it's up to date (preferably whichever version SkyClient staff want you to be on).",
```
Comment (KTibow) [inline crashes.json:1794]:
```suggestion
      "fix": "Issue with Skytils. Make sure it's on a stable version.",
```

File: crashes.json [hunk 3]
```diff
@@ -1241,6 +1687,77 @@
...
+      "causes": [
+        {
+          "method": "contains",
+          "value": "me.SamzyDev"
+        }
+      ]
+    },
+    {
+      "fixtype": 2,
+      "fix": "🚨🚨🚨🚨🚨🚨***__EMERGENCY DETECTED: REMOVE THIS MOD FROM YOUR MODS FOLDER AND GO CHANGE YOUR PASSWORD RIGHT NOW—ASK NO FURTHER QUESTIONS, JUST GO DO IT RIGHT THIS INSTANT. THIS IS FOR THE SAFETY OF YOUR MINECRAFT ACCOUNT.__***🚨🚨🚨🚨🚨🚨 Also, cheaters get banned. (Macro Mod Modules)",
```
Comment (KTibow) [inline crashes.json:1742]:
```suggestion
      "fix": "You have the mod Macro Mod Modules, which is malware and may steal personal info from your computer like passwords and session tokens. PLEASE REMOVE IT FROM YOUR MODS FOLDER, and go CHANGE YOUR PASSWORDS in order to stop others from logging in. Also, cheaters get banned.",
```

File: crashes.json [hunk 4]
```diff
@@ -1055,6 +1116,231 @@
...
+    {
+      "fix": "\"SkyblockChatFilter\" has been detected. Remove it and set Chatting's \"Spam Blocker\" threshold to 95 instead (`-mod chatting`).",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "taj_ny.skyblockchatfilter"
+        }
+      ]
+    },
+    {
```
Comment (KTibow) [inline crashes.json:1317]:
```suggestion
    },
    {
```
i hate to do this to my own feature but sadly @Wyvest is too lazy to make a release that includes the spam blocker

File: crashes.json [hunk 5]
```diff
@@ -1055,6 +1116,231 @@
...
+      "fix": "\"near\" detected. On the fence about Skyclient, hm?",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "studio.dreamys."
+        }
+      ]
+    },
+    {
+      "fix": "\"BetterSkyblockMod\" was found. It has extensive overlap with many features already present in `-mods`, including Skytils, NotEnoughUpdates, and Chatting.",
```
Comment (KTibow) [inline crashes.json:1300]:
```suggestion
      "fix": "\"BetterSkyblockMod\" was found, which has features that other mods originally had, like Skytils, NotEnoughUpdates, and Chatting.",
```

**Comment by KTibow** [comment]
Moment

**Review by UnderscoreRayless — CHANGES_REQUESTED** [review]
- Line 1129-1344 | Add a fixtype, fixtype 2
- I will personally remove all `-mod` messages in your branch.

File: crashes.json [hunk 6]
```diff
@@ -1055,6 +1126,222 @@
...
+      "fix": "reswitch detected. Use NEU's `Dwarven Mines Textures` feature instead, as all necessary Dwarven Mines block textures provided by your resource packs are already loaded on startup this way.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "resourceswitch"
+        }
+      ]
+    },
+    {
+      "fix": "\"near\" detected. On the fence about Skyclient, hm?",
```
Comment (KTibow) [inline crashes.json:1301]: could you explain this

Comment (KTibow) [reply crashes.json:1301]: oh it's a "client" made by some dude who does ratting

File: crashes.json [hunk 7]
```diff
@@ -661,6 +681,27 @@
         }
       ]
     },
+    {
+      "name": "owo",
+      "fix": "To find the solution for SBA's OwO crash, see `-crash sbaowo`. Otherwise, you may be thinking of Kerbybit's MinecwaftOwO, which corresponds to `-crash owomod`.",
+      "causes": [
+        {
+          "method": "regex",
+          "value": "This is a regex test and shouldn't be visible within the Discord server\\."
```
Comment (KTibow) [inline crashes.json:690]: Imagine if some dude says that in game

File: crashes.json [hunk 8]
```diff
@@ -828,14 +869,24 @@
...
       "causes": [
         {
           "method": "contains",
           "value": "sbclient"
         }
       ]
     },
+    {
+      "fixtype": 1,
+      "fix": "Cheaters get banned. Nice SkyClient name ripoff.",
```
Comment (KTibow) [inline crashes.json:882]: Can't you have multiple causes

Comment (RayDeeUx) [reply crashes.json:882]: at this point idk and i cba to check

File: crashes.json (hunk 7 above)
Comment (RayDeeUx) [reply crashes.json:690]: crashlogs ≠ chat logs.

ALSO FUCK I GOTTA REVISE THAT SENTENFE GAAIN

File: crashes.json [hunk 9]
```diff
@@ -189,7 +189,7 @@
     },
     {
       "fixtype": 0,
-      "fix": "You're using Feather Client. Consider not? It's okay to give it up.",
```
Comment (KTibow) [inline crashes.json:182]: HEY @UnderscoreRayless THAT ISNT ERY HUMOR THATS MINE

**Pull request merged**